components/cmake/build_scream.cmake

```diff
@@ -7,8 +7,8 @@ function(build_scream)
 
     # Include machine file here
     message("Found scream component")
-    include(${CMAKE_SOURCE_DIR}/scream/cmake/machine-files/${MACH}.cmake)
-    add_subdirectory("scream")
+    include(${CMAKE_SOURCE_DIR}/eamxx/cmake/machine-files/${MACH}.cmake)
```

Comment:
You might consider renaming this file as well...

Reply:
Renamed.